Fix following warning:

In file included from 3rdparty/heci/heci.h:54,
                 from 3rdparty/heci/heci_main.c:62:
3rdparty/heci/heci_data_structures.h:119:1: warning: "WARN" redefined

Signed-off-by: Herton Ronaldo Krzesinski <herton@mandriva.com.br>

diff -p -up linux-2.6.29/3rdparty/heci/heci_data_structures.h.orig linux-2.6.29/3rdparty/heci/heci_data_structures.h
--- linux-2.6.29/3rdparty/heci/heci_data_structures.h.orig	2009-04-16 03:27:02.000000000 -0300
+++ linux-2.6.29/3rdparty/heci/heci_data_structures.h	2009-04-16 03:27:30.000000000 -0300
@@ -116,7 +116,7 @@
  */
 #define	INFO(format, arg...)	        printk(KERN_INFO     "%s: " format, THIS_MODULE->name, ## arg)
 #define	ERR(format, arg...)	            printk(KERN_ERR      "%s: " format, THIS_MODULE->name, ## arg)
-#define WARN(format, arg...)            printk(KERN_WARNING  "%s: " format, THIS_MODULE->name, ## arg)
+#define WARNI(format, arg...)            printk(KERN_WARNING  "%s: " format, THIS_MODULE->name, ## arg)
 
 
 /* Module Parameters */
